### PR TITLE
[disasters] one geojson call per subject page

### PR DESCRIPTION
**Pull request by chejennifer**
- save geojson data to subject page context & read the context when rendering a disaster event map tile, allowing us to only have to fetch geojson data once

### CODE REVIEW

File: static/js/components/subject_page/data_context.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ * Data context to hold data that is used across multiple components for the
+ * whole page.
+ */
+
+export interface DataContextType {
+  // TODO (chejennifer): use this in map tile
+  geoJsonData: GeoJsonData;
+}
+
+export const DataContext = createContext({} as DataContextType);
```
Comment (beets) [inline static/js/components/subject_page/data_context.ts:31]: nice

File: static/js/components/subject_page/data_context.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+import { GeoJsonData } from "../../chart/types";
+
+/**
+ * Data context to hold data that is used across multiple components for the
+ * whole page.
+ */
+
+export interface DataContextType {
+  // TODO (chejennifer): use this in map tile
```
Comment (beets) [inline static/js/components/subject_page/data_context.ts:27]: update todo to a comment?

**Review by beets — APPROVED** [review]
v cool!

File: static/js/components/subject_page/data_context.ts (hunk 2 above)
Comment (chejennifer) [reply static/js/components/subject_page/data_context.ts:27]: this is already a comment?

**Pull request merged**